packages/atomic/src/components/facets-v1/facet-value-checkbox/facet-value-checkbox.tsx

```diff
@@ -1,14 +1,54 @@
+import {FacetValueState} from '@coveo/headless';
 import {FunctionalComponent, h} from '@stencil/core';
+import {i18n} from 'i18next';
+import {randomID} from '../../../utils/utils';
 
 interface FacetValueCheckboxProps {
+  i18n: i18n;
   value: string;
+  numberOfResults: number;
+  state: FacetValueState;
+  onClick(): void;
 }
 
-// TODO: add interactivity, style & part
 export const FacetValueCheckbox: FunctionalComponent<FacetValueCheckboxProps> = (
   props
-) => (
-  <div>
-    <input type="checkbox" /> {props.value}
-  </div>
-);
+) => {
+  const id = randomID('facet-value-');
```

Comment:
Since it's only used for label/checkbox, I think it's okay and won't negatively affect server side rendering.

Reply:
Yes! :)